quickstarts/newrelic.ingest-metric/dashboards/newrelic.ingest-metric.json

```diff
@@ -0,0 +1,1835 @@
+{
```

Comment:
please run the sanitize script on your submission, there are account IDs, and permissions in this JSON that aren't needed in quickstart.

https://github.com/newrelic/newrelic-quickstarts#-dashboards